### PR TITLE
[strings] Updated Portuguese translations

### PR DESCRIPTION
**Pull request by biodranik**
Thanks to @laralem, see #723

Need to review and fix the strings first, and don't forget to regenerate them before the merge.

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -23765,8 +23887,10 @@
     ko = %d 장소
     no = %d sted
     pl = %d miejsce
-    pt = %d lugar
-    pt-BR = %d lugar
+    pt:one = %s objeto
```
Comment (biodranik) [inline data/strings/strings.txt:23890]: %d here and below

File: data/strings/strings.txt [hunk 2]
```diff
@@ -23804,8 +23928,10 @@
     ko = %d 장소들
     no = %d steder
     pl = %d miejsc
-    pt = %d lugares
-    pt-BR = %d lugares
+    pt:one = %s lugar
```
Comment (biodranik) [inline data/strings/strings.txt:23931]: %d here and below

File: data/strings/strings.txt [hunk 3]
```diff
@@ -23843,8 +23969,11 @@
     ko = %d 추적
     no = %d stier
     pl = %d tras
-    pt = %d rastreamentos
-    pt-BR = %d cursos
+    pt:one = %s percurso
```
Comment (biodranik) [inline data/strings/strings.txt:23972]: %d here and below

File: data/strings/strings.txt [hunk 4]
```diff
@@ -24670,6 +24807,8 @@
 
   [why_support_item1]
     en = We respect your privacy: there are zero trackers in the app, and we are open source
+    pt = Respeitamos a sua privacidade: não existem rastreadores na aplicação e o código-fonte é aberto
```
Comment (biodranik) [inline data/strings/strings.txt:24810]: We historically start translations from en and ru, as our team can read them easily and check/add other (ideally sorted) translations below based on that.

File: data/strings/strings.txt [hunk 5]
```diff
@@ -5849,8 +5860,8 @@
     ko = 우리는 Organic Maps. Lite에서 Organic Maps로 다운로드된 지도를 전송하고 있습니다. 이는 몇 분 정도 걸릴 수 있습니다.
     nb = Vi overfører dine nedlastede kart fra Organic Maps Lite til Organic Maps. Det kan ta noen minutter.
     pl = Przenosimy twoje pobrane mapy z Organic Maps Lite do Organic Maps. Może to zająć kilka minut.
-    pt = Estamos a transferir os seus mapas descarregados da Organic Maps Lite para a Organic Maps. Isto pode demorar alguns minutos.
-    pt-BR = Estamos transferindo os seus mapas baixados do Organic Maps Lite para o Organic Maps. Isto pode demorar alguns minutos.
```
Comment (biodranik) [inline data/strings/strings.txt:5853]: [move_lite_maps_to_pro]
[move_lite_maps_to_pro_ok]
[move_lite_maps_to_pro_failed]
These translated blocks should be completely removed, they are not used anymore for a long time.

File: data/strings/strings.txt (hunk 1 above)
Comment (vng) [reply data/strings/strings.txt:23890]: pt:one - это так и надо?

File: data/strings/strings.txt (hunk 4 above)
Comment (vng) [reply data/strings/strings.txt:24810]: Тут pt, а выше было pt:one
Стоит сделать единообразно

File: data/strings/strings.txt (hunk 1 above)
Comment (biodranik) [reply data/strings/strings.txt:23890]: Это перевод для единственного числа аргумента. Twine так умеет )

File: data/strings/strings.txt (hunk 4 above)
Comment (biodranik) [reply data/strings/strings.txt:24810]: Не, one это когда параметр равен 1.

**Comment by Suburbanno** [comment]
hope to see this soon on android

**Comment by biodranik** [comment]
@Suburbanno can you please check that the Brazilian dialect is properly working now on Android? E.g. that there are some different translations for Portugal and Brazilian versions.

**Comment by Suburbanno** [comment]
> @Suburbanno can you please check that the Brazilian dialect is properly working now on Android? E.g. that there are some different translations for Portugal and Brazilian versions.

I need an updated build, can you send me? I don't have the necessary means to build...

**Comment by biodranik** [comment]
@Suburbanno Can you please confirm that the current build at least switches to Brazilian for some text elements in the interface?

Also,  it would be great if someone can take a look at [data/strings/types_strings.txt](../blob/master/data/strings/types_strings.txt) and [iphone/plist.txt](../blob/master/iphone/plist.txt) and add/fix translations there.

**Comment by Suburbanno** [comment]
> @Suburbanno Can you please confirm that the current build at least switches to Brazilian for some text elements in the interface?
> 
> Also,  it would be great if someone can take a look at [data/strings/types_strings.txt](../blob/master/data/strings/types_strings.txt) and [iphone/plist.txt](../blob/master/iphone/plist.txt) and add/fix translations there.

I'm on version 2021.06.23-9, it's still shown in Portuguese from Portugal.  I checked the files and they really need translations, I will do a PR soon...

File: iphone/Maps/LocalizedStrings/ro.lproj/Localizable.strings [hunk 6]
```diff
@@ -1548,7 +1539,7 @@
 
 "off" = "Dezactivat";
 
-"prefs_languages_information" = "Pentru instrucțiuni vocale utilizăm sistemul TTS. Multe dispozitive cu Android folosesc Google TTS. Puteți descărca sau actualiza aplicația din Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)";
+"prefs_languages_information" = "We use system TTS for voice instructions. Many Android devices use Google TTS, you can download or update it from Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)";
```
Comment (vng) [inline iphone/Maps/LocalizedStrings/ro.lproj/Localizable.strings:1542]: Why translation become English?

File: data/strings/strings.txt [hunk 7]
```diff
@@ -3253,6 +3254,8 @@
 
   [share_bookmarks_email_body]
     en = Hello!\n\nAttached are my bookmarks from Organic Maps app. Please open them if you have Organic Maps installed. Or, if you don't, download the app for your iOS or Android device by following this link: https://omaps.app/get?kmz\n\nEnjoy traveling with Organic Maps!
+    pt = Olá!\n\nSegue em anexo os meus favoritos da aplicação Organic Maps. Por favor abra-os se tiver o Organic Maps instalado. Caso não tenha, descarregue a aplicação para o seu dispositivo iOS ou Android com a hiperligação https://omaps.app/get?kmz\n\nDivirta-se a viajar com o Organic Maps!
+    pt = Olá!\n\nSegue em anexo meus favoritos do app Organic Maps. Por favor os abra se você tiver o Organic Maps instalado. Caso não tenha, baixe o app para o seu dispositivo iOS ou Android com o link https://omaps.app/get?kmz\n\nDisfrute viajar com o Organic Maps!
```
Comment (vng) [inline data/strings/strings.txt:3258]: Why pt 2 times here?

File: data/strings/strings.txt [hunk 8]
```diff
@@ -13497,7 +13408,7 @@
     ko = 등록
     nb = Registrer deg
     pl = Zarejestruj się
-    pt = Cadastrar-se
+    pt = Crie uma conta no OpenStreetMap
```
Comment (vng) [inline data/strings/strings.txt:13411]: "OpenStreetMap" is not needed here?

File: data/strings/strings.txt [hunk 9]
```diff
@@ -20288,8 +20264,9 @@
...
-    pt = Utilizamos o sistema TTS para instruções de voz. Muitos dispositivos Android usam o Google TTS, pode transferir ou atualizá-lo a partir do Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)
-    ro = Pentru instrucțiuni vocale utilizăm sistemul TTS. Multe dispozitive cu Android folosesc Google TTS. Puteți descărca sau actualiza aplicația din Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)
+    pt = Utilizamos o sistema TTS para as instruções de voz. Muitos dispositivos Android usam o Google TTS, pode transferir ou atualizá-lo a partir do Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)
+    pt-BR = Utilizamos o sistema TTS para instruções de voz. Muitos dispositivos Android usam o Google TTS, pode transferir ou atualizá-lo a partir do Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)
+    rro = Pentru instrucțiuni vocale utilizăm sistemul TTS. Multe dispozitive cu Android folosesc Google TTS. Puteți descărca sau actualiza aplicația din Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)
```
Comment (vng) [inline data/strings/strings.txt:20269]: ro should remain unchnaged

**Review by vng — CHANGES_REQUESTED** [review]

**Review by vng — APPROVED** [review]

**Pull request merged**